tasks/uwsgi.yml

```diff
@@ -0,0 +1,6 @@
+# set directory permissions for uswgi for logs and pid file
+- file:
```

Comment:
I found out this is not necessary because is already being performed here:
https://github.com/openwisp/ansible-openwisp2/blob/master/tasks/django.yml#L13-L18

I tested your patch without this task and it works anyway, so better remove this file and the related include in directive in `tasks/main.yml`, the rest is fine, thank you @xamfy!